### PR TITLE
chore: update product references

### PR DESCRIPTION
**Pull request by mwickett**
- chore: update product metadata
- chore: update product nomenclature

## 🗒️ What

Updates some nomenclature around products.

## 🤷 Why

See https://www.hashicorp.com/license-faq for more detail.

### CODE REVIEW

File: src/components/tutorials-landing-view/docs.mdx [hunk 1]
```diff
@@ -53,13 +53,13 @@ peerComponents:
...
 			description:
-				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform open source.',
+				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community.',
 		},
 		{
 			slug: 'networking-automation',
 			title: 'Verify your Networking Automation skills',
 			description:
-				'The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain open source HashiCorp Consul.',
+				'The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain HashiCorp Consul Community.',
```
Comment (kendallstrautman) [inline src/components/tutorials-landing-view/docs.mdx:62]:
```suggestion
				'The Consul Associate Certification is for Site Reliability Engineers, Solutions Architects, DevOps professionals, or other Cloud Engineers who know the basic concepts and skills to build, secure, and maintain HashiCorp Consul Community Edition.',
```

File: src/components/tutorials-landing-view/docs.mdx [hunk 2]
```diff
@@ -53,13 +53,13 @@ peerComponents:
 			slug: 'infrastructure-automation',
 			title: 'Verify your Infrastructure Automation skills',
 			description:
-				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform open source.',
+				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community.',
```
Comment (kendallstrautman) [inline src/components/tutorials-landing-view/docs.mdx:56]:
```suggestion
				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community Edition.',
```

File: src/content/boundary/product-landing.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
 			},
 			{
 				"icon": "hcp",
-				"text": "Boundary OSS Quick Start",
+				"text": "Boundary Community Quick Start",
```
Comment (kendallstrautman) [inline src/content/boundary/product-landing.json:29]: This is associated with an actual collection title so I'd be hesitant to make this change.

Comment (mwickett) [reply src/content/boundary/product-landing.json]: ok, removed it for now.

File: src/content/packer/product-landing.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	},
 	"overview": {
 		"heading": "What is Packer?",
-		"body": "Packer is an open source tool that lets you create identical machine images for multiple platforms from a single source template. Packer can create golden images to use in image pipelines.",
+		"body": "Packer is a tool that lets you create identical machine images for multiple platforms from a single source template. Packer can create golden images to use in image pipelines.",
```
Comment (kendallstrautman) [inline src/content/packer/product-landing.json:8]: We can merge this but ideally education would rework this sentence/ approve it.

File: src/content/certifications/programs/infrastructure-automation.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
 			"examCode": "003",
 			"productSlug": "terraform",
 			"versionTested": "Terraform 1.0 or higher",
-			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform open source. This includes understanding which enterprise features exist and what can and cannot be done using the open source offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
+			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community. This includes understanding which enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```
Comment (kendallstrautman) [inline src/content/certifications/programs/infrastructure-automation.json:17]:
```suggestion
			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community Edition. This includes understanding which enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```

File: src/content/certifications/programs/security-automation.json [hunk 6]
```diff
@@ -2,19 +2,19 @@
 	"title": "Security Automation",
 	"summary": {
 		"heading": "Security<br />Automation Certifications",
-		"description": "Earn the Vault Associate to validate your knowledge of the concepts, skills, and use cases associated with open source Vault. Ready to demonstrate your advanced Vault Enterprise skills in production? Successfully complete a lab-based exam and earn the Vault Operations Professional certification."
+		"description": "Earn the Vault Associate to validate your knowledge of the concepts, skills, and use cases associated with Vault Community. Ready to demonstrate your advanced Vault Enterprise skills in production? Successfully complete a lab-based exam and earn the Vault Operations Professional certification."
```
Comment (kendallstrautman) [inline src/content/certifications/programs/security-automation.json:5]:
```suggestion
		"description": "Earn the Vault Associate to validate your knowledge of the concepts, skills, and use cases associated with Vault Community Edition. Ready to demonstrate your advanced Vault Enterprise skills in production? Successfully complete a lab-based exam and earn the Vault Operations Professional certification."
```

File: src/content/certifications/programs/security-automation.json [hunk 7]
```diff
@@ -2,19 +2,19 @@
...
 	},
 	"hero": {
 		"heading": "Security<br />Automation Certification",
-		"description": "HashiCorp’s Security Automation certification program has two levels: Work up to the advanced Vault Professional Certification by starting with the foundational Vault Associate certification. The Associate certification validates your knowledge of open source Vault. Then, continue your certification journey with the Professional hands-on, lab-based exam to validate your years of production experience with both Vault and Vault Enterprise."
+		"description": "HashiCorp’s Security Automation certification program has two levels: Work up to the advanced Vault Professional Certification by starting with the foundational Vault Associate certification. The Associate certification validates your knowledge of Vault Community. Then, continue your certification journey with the Professional hands-on, lab-based exam to validate your years of production experience with both Vault and Vault Enterprise."
```
Comment (kendallstrautman) [inline src/content/certifications/programs/security-automation.json:9]:
```suggestion
		"description": "HashiCorp’s Security Automation certification program has two levels: Work up to the advanced Vault Professional Certification by starting with the foundational Vault Associate certification. The Associate certification validates your knowledge of Vault Community Edition. Then, continue your certification journey with the Professional hands-on, lab-based exam to validate your years of production experience with both Vault and Vault Enterprise."
```

File: src/content/certifications/programs/security-automation.json [hunk 8]
```diff
@@ -2,19 +2,19 @@
...
 	},
 	"exams": [
 		{
 			"title": "Vault Associate",
 			"examCode": "002",
 			"productSlug": "vault",
 			"versionTested": "Vault 1.6.0 and higher",
-			"description": "The Vault Associate certification is for Cloud Engineers specializing in security, development, or operations who know the basic concepts, skills, and use cases associated with open source Vault. This includes understanding what enterprise features exist and what can and cannot be done using the open source offering. You should have professional experience using Vault in production, but performing the exam objectives in a personal demo environment may be sufficient.",
+			"description": "The Vault Associate certification is for Cloud Engineers specializing in security, development, or operations who know the basic concepts, skills, and use cases associated with Vault Community. This includes understanding what enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Vault in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```
Comment (kendallstrautman) [inline src/content/certifications/programs/security-automation.json:17]:
```suggestion
			"description": "The Vault Associate certification is for Cloud Engineers specializing in security, development, or operations who know the basic concepts, skills, and use cases associated with Vault Community Edition. This includes understanding what enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Vault in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```

**Review by kendallstrautman — CHANGES_REQUESTED** [review]

File: src/content/packer/product-landing.json (hunk 4 above)
Comment (mwickett) [reply src/content/packer/product-landing.json:8]: I think we're fine to go ahead here, and have them look at it after the fact.

File: src/data/packer.json [hunk 9]
```diff
@@ -12,7 +12,7 @@
 	"datoToken": "88b4984480dad56295a8aadae6caad",
 	"metadata": {
 		"title": "Packer by HashiCorp",
-		"description": "Packer is a free and open source tool for creating golden images for multiple platforms from a single source configuration.",
+		"description": "Packer is a tool for creating golden images for multiple platforms from a single source configuration.",
```
Comment (im2nguyen) [inline src/data/packer.json:15]:
```suggestion
		"description": "Packer helps you create golden images for multiple platforms from a single source configuration.",
```

**Review by im2nguyen — APPROVED** [review]
looks good on my end, thank you!

**Review by kendallstrautman — APPROVED** [review]

**Review by yhyakuna — APPROVED** [review]
LGTM

File: src/content/certifications/programs/infrastructure-automation.json [hunk 10]
```diff
@@ -14,7 +14,7 @@
 			"examCode": "003",
 			"productSlug": "terraform",
 			"versionTested": "Terraform 1.0 or higher",
-			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform open source. This includes understanding which enterprise features exist and what can and cannot be done using the open source offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
+			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community Edition. This includes understanding which enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```
Comment (ritsok) [inline src/content/certifications/programs/infrastructure-automation.json:17]:
```suggestion
			"description": "The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform. This includes understanding which enterprise features exist and what can and cannot be done using the community offering. You should have professional experience using Terraform in production, but performing the exam objectives in a personal demo environment may be sufficient.",
```

File: src/content/certifications/exam-faqs/terraform-associate-002.mdx [hunk 11]
```diff
@@ -58,20 +58,20 @@
...
 | 8e    | Use resource addressing and resource parameters to connect resources together                                       |
 | 8f    | Use Terraform built-in functions to write configuration                                                             |
-| 8g    | Configure resource using a `dynamic` block                                                                            |
+| 8g    | Configure resource using a `dynamic` block                                                                          |
 | 8h    | Describe built-in dependency management (order of execution based)                                                  |
 | **9** | **Understand Terraform Cloud and Enterprise capabilities**                                                          |
 | 9a    | Describe the benefits of Sentinel, registry, and workspaces                                                         |
-| 9b    | Differentiate OSS and Terraform Cloud workspaces                                                                    |
+| 9b    | Differentiate Terraform Community and Terraform Cloud workspaces                                                    |
```
Comment (ritsok) [inline src/content/certifications/exam-faqs/terraform-associate-002.mdx:74]:
```suggestion
| 9b    | Differentiate Terraform and Terraform Cloud workspaces                                                    |
```

File: src/components/tutorials-landing-view/docs.mdx [hunk 12]
```diff
@@ -53,13 +53,13 @@ peerComponents:
 			slug: 'infrastructure-automation',
 			title: 'Verify your Infrastructure Automation skills',
 			description:
-				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform open source.',
+				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform Community Edition.',
```
Comment (ritsok) [inline src/components/tutorials-landing-view/docs.mdx:56]:
```suggestion
				'The Terraform Associate certification is for Cloud Engineers specializing in operations, IT, or development who know the basic concepts and skills associated with Terraform.',
```

**Review by ritsok — APPROVED** [review]
removing community edition in a few places. tf associate exam still (for now) uses a version that's under oss, don't want to cause confusion.

will revisit phrasing after discussing in more detail with certs team.

**Pull request merged**